### DISCUSSION TITLE
redis UID (security context)

### DISCUSSION THREAD

**Discussion by LeFox21** [argoproj-labs/argocd-operator#1573] [Q&A] [answered]
Hello,
I see in the deployment.go code that UID 999 is hardcoded for redis, is there any particular reason why this was hardcoded ? I'm actually facing deployment issues as my PSP are refusing UID 999 on my clusters.
![image](https://github.com/user-attachments/assets/7af949b6-cc83-4180-a679-2d7d94ecde94)

**Comment by svghadi** [comment, marked as answer]
Hi @LeFox21, we just merged a PR to update this uid to `1000`. Hope that works for you. The change will be available in upcoming 0.14.0 release. 

For more context: https://github.com/argoproj-labs/argocd-operator/issues/1662

I am closing this discussion. Please feel free to reopen if `uid: 1000` doesn't work for you.
